x11/windowmaker/Portfile

```
@@ -38,26 +48,20 @@ depends_lib \
 	port:giflib \
 	port:tiff
 
-post-patch {
-	set fl [glob -directory ${worksrcpath} WindowMaker/*menu*]
-	lappend fl ${worksrcpath}/WindowMaker/Defaults/WindowMaker.in \
-	               ${worksrcpath}/WPrefs.app/Menu.c ${worksrcpath}/WPrefs.app/Paths.c
-	foreach i $fl {
-		reinplace "s|/usr/X11R6/share|${prefix}/share|g" $i
-		reinplace "s|/usr/share|${prefix}/share|g" $i
-		reinplace "s|/usr/local/share|${prefix}/share|g" $i
-		reinplace "s|/opt/share|${prefix}/share|g" $i
-		reinplace "s|/opt/kde2/share|${prefix}/share|g" $i
-	}
-}
+configure.env   LINGUAS="bg cs da de el es et fi fr gl hr hu it ja \
+                ko ms nl no pl pt ro ru sk sv tr zh_CN"
```

This `configure.env` change reverts a560834f1e38aeec9e34b708baaed09366fa68e0 / 88ea58499404818b508cb19e6064716e2d2c9317, breaking compatibility with MacPorts 2.6.0 and later. There are warnings in the compile log caused by this. This change should be undone.
```suggestion
configure.env   "LINGUAS=bg cs da de el es et fi fr gl hr hu it ja \
                ko ms nl no pl pt ro ru sk sv tr zh_CN"
```